src/api/test/functional/build_controller_test.rb

```diff
@@ -156,12 +156,17 @@ def test_buildinfo
 
     # osc local package build call
     get "/source/home:Iggy/TestPack/TestPack.spec"
+    assert_response :success
     raw_post "/build/home:Iggy/10.2/i586/_repository/_buildinfo", @response.body
     assert_response :success
 
     # this is only testing the rep server buildinfo, not the one generated by scheduler
     get "/build/home:Iggy/10.2/i586/TestPack/_buildinfo"
     assert_response :success
+    rev="12"
+    b_cnt="1"
+    ci_cnt="52"
```

Comment:
yes, because we do not have a real rollback of revisions for this package, so it is currently likely that you need either one or the other numbers to debug